hooks/src/index.js

```diff
@@ -250,11 +250,24 @@ export function useContext(context) {
 	// This is probably not safe to convert to "!"
 	if (state._value == null) {
```

Comment:
This might be smaller now:

```suggestion
	if (!state. _subscription) {
```